app/models/manageiq/providers/ibm_terraform/configuration_manager/configured_system.rb

```diff
@@ -22,6 +40,10 @@ def console_url
     end
   end
 
+  def label_for_vendor_icon
```

Comment:
I guess this answers my [question in the decorators repo](https://github.com/ManageIQ/manageiq-decorators/pull/37#discussion_r508280574). Having the logic on two different places can make this a little bit hard to follow. You can move the decorator into this repo and have the constant and the icon selection logic in-place. 

I linked an example in my comment of how the AutoSDE folks have done this...

Reply:
Thanks @skateman. I created a provider specific decorator.